src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -559,15 +559,15 @@ export class ExtensionsListView extends ViewsViewletPanel {
 
 export class InstalledExtensionsView extends ExtensionsListView {
 
-	public static isInsalledExtensionsQuery(query: string): boolean {
+	public static isInstalledExtensionsQuery(query: string): boolean {
 		return ExtensionsListView.isInstalledExtensionsQuery(query)
 			|| ExtensionsListView.isOutdatedExtensionsQuery(query)
 			|| ExtensionsListView.isDisabledExtensionsQuery(query)
 			|| ExtensionsListView.isEnabledExtensionsQuery(query);
 	}
 
 	async show(query: string): TPromise<IPagedModel<IExtension>> {
-		if (InstalledExtensionsView.isInsalledExtensionsQuery(query)) {
```

Comment:
Just stumbled on this as well, I was confused why pressing `⌘+D` selected only first `Installed` on this line but not the 2nd, turned out the 2nd has a typo.